### DISCUSSION TITLE
Integrazione Netatmo

### DISCUSSION THREAD

**Discussion by Matty1184** [indomus/forum#2165] [GENERALE - Discussioni generiche]
Ciao a tutti,
volevo installare l'integrazione netatmo ma da quello che capisco non si connette a netatmo
```
Logger: homeassistant.components.netatmo
Source: components/netatmo/__init__.py:239
Integration: Netatmo ([documentation](https://www.home-assistant.io/integrations/netatmo), [issues](https://github.com/home-assistant/home-assistant/issues?q=is%3Aissue+is%3Aopen+label%3A%22integration%3A+netatmo%22))
First occurred: 08:48:30 (1 occurrences)
Last logged: 08:48:30
Error during webhook registration - 503 - Service Unavailable - when accessing 'https://api.netatmo.com/api/addwebhook'
```
```
2022-12-12 08:48:30.861 ERROR (MainThread) [homeassistant.components.netatmo] Error during webhook registration - 503 - Service Unavailable - when accessing 'https://api.netatmo.com/api/addwebhook'
```

ho insertito nel configuration.yaml la riga con i parametri dell'app creata su netatmo web

```
netatmo:
  client_id: xxxxxxxx
  client_secret: xxxxxxxxx
```

dove sto sbagliando?

**Comment by indomus** [comment]
> Ciao a tutti, volevo installare l'integrazione netatmo ma da quello che capisco non si connette a netatmo
> 
> ```
> Logger: homeassistant.components.netatmo
> Source: components/netatmo/__init__.py:239
> Integration: Netatmo ([documentation](https://www.home-assistant.io/integrations/netatmo), [issues](https://github.com/home-assistant/home-assistant/issues?q=is%3Aissue+is%3Aopen+label%3A%22integration%3A+netatmo%22))
> First occurred: 08:48:30 (1 occurrences)
> Last logged: 08:48:30
> Error during webhook registration - 503 - Service Unavailable - when accessing 'https://api.netatmo.com/api/addwebhook'
> ```
> 
> ```
> 2022-12-12 08:48:30.861 ERROR (MainThread) [homeassistant.components.netatmo] Error during webhook registration - 503 - Service Unavailable - when accessing 'https://api.netatmo.com/api/addwebhook'
> ```
> 
> ho insertito nel configuration.yaml la riga con i parametri dell'app creata su netatmo web
> 
> ```
> netatmo:
>   client_id: xxxxxxxx
>   client_secret: xxxxxxxxx
> ```
> 
> dove sto sbagliando?

non stai sbagliando. Netatmo in questi giorni ha qualche problema transitorio.

**Comment by Matty1184** [comment]
ok grazie,  alternativa Homekit o bipassare collegando i dispositivi a zigbee giusto?

**Reply by Scialla** [reply]
Puoi integrarlo tramite homekit, ma ha meno attributi però è più reattivo, puoi integrarlo in entrambi i modi. Cmq quell'errore lo da per webhook (anche a me) e non influisce sull'integrazione, funziona tutto lo stesso. Zigbee con Netatmo non c'entra proprio una fava

**Reply by Matty1184** [reply]
onestamente volevo evitare Homekit e come zigbee intedevo bipassare proprio gateway e app bticino e collegare i dispositivi a zigbee direttamente in HA.
mi scrivi che l'errore è solo per webhook ma non sull'integrazione, quindi i dispositivi dovrei vederli comunque??

**Reply by Scialla** [reply]
ma di quali dispositivi zigbee parli? Netatmo non fa dispositivi zigbee....
Si i dispositivi si vedono tranquillamente anche con l'errore Webhook (di solito il 400 credo) ma, tieni conto, che il sito di Netatmo sta avendo dei problemi in questi giorni. Io ho appena riavviato HA (per altri motivi) e ho errore `Error during webhook registration - Webhook registration timed out` ma vedo e piloto correttamente termostato e valvole

**Reply by Matty1184** [reply]
scusa avevo omesso il tipo di oggetto che volevo integrare...
io vorrei integrare alcuni switch luci e controlli tapparelle della serie bticino smart. ho letto che sono tutti gestiti da netatmo e quindi stavo cercando di integrarli senza homekit

**Reply by Scialla** [reply]
scusa ma l'integrazione netatmo "integra" tutto ciò che hai registrato sul portale di netatmo. Non mi risulta che sul portale netatmo si registrino i dispositivi bticino, ma magari sbaglio. Accertati prima di questo

**Reply by Matty1184** [reply]
dalla pagina ufficiale io capisco di si.
https://www.home-assistant.io/integrations/netatmo/
temo però di sbagliare qualcosa nella configurazione in quanto le è come dici tu le entità doveva trovarle tutte comunque.

**Reply by Scialla** [reply]
leggi bene, da nessuna parte parla di bticino... leggi cosa dice nelle varie voci, soprattutto light e switch, sono entità esposte dalla camera e dalla weather station

**Reply by Matty1184** [reply]
perfetto.. grande abbaglio che ho preso... grazie mille perchè 'avevo proprio interpretato in modo errato.

**Reply by Matty1184** [reply]
> leggi bene, da nessuna parte parla di bticino... leggi cosa dice nelle varie voci, soprattutto light e switch, sono entità esposte dalla camera e dalla weather station

Ho trovato delle delucidazioni e guide e sul sito legrand, dove spiega che Home+Control viene sostituito da Netatmo Connect tramite autenticazione dedicata e tutti i componenti da luci, dimmer, interruttori, tapparelle saranno gestiti da li

**Reply by indomus** [reply]
> > leggi bene, da nessuna parte parla di bticino... leggi cosa dice nelle varie voci, soprattutto light e switch, sono entità esposte dalla camera e dalla weather station
> 
> Ho trovato delle delucidazioni e guide e sul sito legrand, dove spiega che Home+Control viene sostituito da Netatmo Connect tramite autenticazione dedicata e tutti i componenti da luci, dimmer, interruttori, tapparelle saranno gestiti da li

https://indomus.it/news/home-project-per-gli-impianti-myhome-di-bticino-tante-novita-e-qualche-problema-di-gioventu/
